src/constants.js

```diff
@@ -0,0 +1,5 @@
+const LIKE_BAR_MIN_POWER = 125;
```

Comment:
Instead of creating a new file we can add this constant into the root file `config.json`.